README.md

```diff
@@ -60,20 +60,5 @@ simply a place holder for names given to future implementations.
 
 # Contributing
 
-Contributors who wish to provide an alternative implementation of `yppm`
-must do so by issuing a Pull Request to the `develop` branch from their
-own forks of this repository. Write access to this repository can not be
-granted.
-
-All pull requests to this repository must include:
-
-* A build system that functions across platforms and (if applicable) across
-multiple compilers.
-* Source code that is well written, properly formatted, and documented.
-* A test suite that passes and duplicates the tests that are run in the
-reference implementation. At present, 11 digits of accuracy are expected.
```

Comment:
The yppm kernel is 32-bit floating point, -real-size 32. 11 digits of accuracy is too many for yppm.  4-5 digits of accuracy is more reasonable.  Here is what we are currently testing against.

EST        Variable                 Min                 Max               First                Last                 RMS
TEST -------------------------------------------------------------------------------------------------------------------
TEST             fy2        0.877492E-05        0.502973E-04        0.314929E-04        0.251612E-04        0.285662E-04
TEST               q        0.111311E+04        0.111311E+04        0.111311E+04        0.111311E+04        0.111311E+04
TEST             cry       -0.252956E-03        0.126737E-02        0.469859E-03        0.651557E-03        0.698602E-03
TEST             dya        0.293576E+04        0.300913E+04        0.298094E+04        0.295948E+04        0.297277E+04

Reply:
Indeed.  Good catch!

Reply:
Fixed.